Userland/Utilities/image.cpp

```diff
@@ -66,6 +69,20 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         return 1;
     }
 
+    if (dump_metadata) {
```

Comment:
I don't think this should be in `image`. We should have an `exif` tool that does this.